packages/data-connect/src/api/index.ts

```diff
@@ -22,3 +22,10 @@ export * from './Mutation';
 export * from './query';
 export { setLogLevel } from '../logger';
 export { validateArgs } from '../util/validateArgs';
+export {
+  DataConnectErrorCode,
+  DataConnectError,
+  DataConnectOperationError,
+  DataConnectOperationResponse,
```

Comment:
Please rename DataConnectOperationResponse to DataConnectOperationFailureResponse as per go/dataconnect:sdk:partial-errors